doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -474,6 +474,17 @@ arguments used for configuration.  Here is a table describing these emulated dev
           latter is ignored and the MAC address is set to the ``mac`` value.
           ``mac_seed`` will only be used when ``mac`` is not set.
 
+   * - ``virtio-gpu``
+     - Virtio GPU type device, parameters format is:
+       ``virtio-gpu[,geometry=<width>x<height>+<x_off>+<y_off> | fullscreen:<monitor_id>]``
+
+       * ``<geometry>`` specifies the mode of virtual display, windowed or fullscreen
+       * ``width`` specifies the width of the virtual display window in pixels.
+       * ``height`` specifies the height of the virtual display window in pixels.
+       * ``x_off`` specifies the x offfset of the virtual display window from the upper-left corner of screen.
+       * ``y_off`` specifies the y offfset of the virtual display window from the upper-left corner of screen.
+       * ``monitor_id`` specifies the screen id when show virtual display as fullscreen, eg.0,1,2....
```

Comment:
```suggestion
       * ``monitor_id`` specifies the screen id when the virtual display is fullscreen, e.g., 0,1,2.
       
       When used, these width and height values are separated by an ``x``,
       and the x_off and y_off values are separated by a ``+``. For example:
       ``geometry=1280x720+100+50`` specifies a window 1280 pixels wide
       by 720 high, and 100 pixels right and 50 pixels down from the top left
       corner of the screen.
```
Can the user specify ``0xFFFF`` (or 65535) for the two offset values to specify  the "don't care" SDL_WINDOWPOS_UNDEFINED value, and can we document that?